README.md

```diff
@@ -1,68 +1,6 @@
-# Eppo Rust SDK
```

Comment:
This is the top level README is probably largely ok - however we can do another pass at all this separately.